tests/system/test_client.py

```diff
@@ -2173,6 +2173,35 @@ def test_list_rows_page_size(self):
         page = next(pages)
         self.assertEqual(page.num_items, num_last_page)
 
+    def test_parameterized_types_round_trip(self):
+        client = Config.CLIENT
+        table_id = f"{Config.DATASET}.test_parameterized_types_round_trip"
+        fields = (
+            ("n", "NUMERIC"),
+            ("n9", "NUMERIC(9)"),
+            ("n92", "NUMERIC(9, 2)"),
+            ("bn", "BIGNUMERIC"),
+            ("bn9", "BIGNUMERIC(38)"),
+            ("bn92", "BIGNUMERIC(38, 22)"),
+            ("s", "STRING"),
+            ("s9", "STRING(9)"),
+            ("b", "BYTES"),
+            ("b9", "BYTES(9)"),
+        )
+        client.query(
+            "create table {} ({})".format(
+                table_id, ", ".join(" ".join(f) for f in fields)
+            )
+        ).result()
+        table = client.get_table(table_id)
+        self.to_delete.insert(0, table)
+        table_id2 = table_id + "2"
+        client.create_table(Table(f"{client.project}.{table_id2}", table.schema))
+        table2 = client.get_table(table_id2)
+        self.to_delete.insert(0, table2)
```

Comment:
Just above this line, `client.create_table()` makes an API request, and if it fails `table2` will not be added to the `to_delete` list. It seems we need this one line up?

Reply:
Done.

Note that the dataset is deleted, so this is fairly moot.

Reply:
Right... so the worst it could happen is that a table existed for a bit longer than necessary. Tunnel vision I'd say.